angular_vel_cov:
  - 4.3529351987328976e-05
  - 0.0
  - 0.0
  - 0.0
  - 4.43231889048116e-05
  - 0.0
  - 0.0
  - 0.0
  - 2.2526434315870354e-05
linear_accel_cov:
  - 0.0005466110186805089
  - 0.0
  - 0.0
  - 0.0
  - 0.0010596120668389785
  - 0.0
  - 0.0
  - 0.0
  - 0.0018152782842159802
orientation_cov:
  - 3.535376710125298e-08
  - 0.0
  - 0.0
  - 0.0
  - 7.776062425308101e-08
  - 0.0
  - 0.0
  - 0.0
  - 1.473859432098556e-07
